code/modules/emotes/emote_mob.dm

```diff
@@ -168,6 +168,7 @@
 	message = trim(html_encode(message))
 	message = filter_modify_message(message)
 	message = format_emote(src, message)
+	message = trim(html_decode(message))
```

Comment:
I forget why it was encoded in the first place, I swear that at one point it was decoded after, but I can't find it now. Anyway, this trim had to be added regardless. I added trims to the markdown filters but I don't necessarily trust them to be exhaustive.